fem/bilininteg.cpp

```diff
@@ -1925,7 +1998,7 @@ void VectorFEMassIntegrator::AssembleElementMatrix2(
    }
 }
 
-void VectorDivergenceIntegrator::AssembleElementMatrix2(
+void DivergenceIntegrator::AssembleElementMatrix2(
```

Comment:
This name change will break backwards compatibility (although I agree it's a better name)